cosmos/pocket.json

```diff
@@ -1,9 +1,10 @@
 {
-  "chainId": "poktroll",
+
+  "chainId": "pocket",
   "chainName": "Pocket Network",
-  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/poktroll/chain.png",
-  "rpc": "https://shannon-testnet-grove-seed-rpc.poktroll.com",
-  "rest": "https://shannon-testnet-grove-seed-api.poktroll.com",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/pocket/chain.jpg",
```

Comment:
```suggestion
  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/pocket/chain.png",
```

jpg -> png

Also, the directory where the `chain.png` file is located should be renamed from `pocket-mainnet` to `pocket`.
<img width="623" alt="Screenshot 2025-04-02 at 3 24 28 PM" src="https://github.com/user-attachments/assets/fd6c06ef-763a-4c00-ab3a-d7b6b8c33b6c" />

This is the file we're referencing here.